app/src/pages/outside/loginPage/loginBlock/loginForm/externalLoginBlock/externalLoginBlock.jsx

```diff
@@ -31,14 +32,18 @@ export const ExternalLoginBlock = ({ externalAuth }) => (
   <div className={cx('external-login-block')}>
     {Object.keys(externalAuth).map((objKey) => {
       const val = externalAuth[objKey];
+      let path = val.path;
+      if (path.indexOf(UAT_API_URL_PREFIX) === -1) {
+        path = `/uat${val.path}`;
```

Comment:
I think it's better to use `UAT_API_URL_PREFIX` here too

Reply:
Fixed